templates/haskell/.stylish-haskell.yaml

```diff
@@ -18,9 +18,6 @@ steps:
       list_padding: 2
       separate_lists: false
       space_surround: false
-  - language_pragmas:
-      style: compact
-      remove_redundant: true
```

Comment:
We can comment it out and the let project initiator decide what they want.
```
# If you don't want language pragmas to be affected, remove these lines.
# Otherwise, set style to `compact`, `vertical` or `compact_line` as you prefer.
#  - language_pragmas:
#      style: compact
#      remove_redundant: true
```

So when you create a new repo, you will decide what to do. Files in this repo are _templates_ and some of them require some edits before you can use them. Is it an acceptable solution? At least it should solve this problem:
> (I have to find some another config with this option set and copy-paste it, or type manually which is even sadder

Reply:
Yeah, sounds like the best solution. Then I think it's worth making sure that this commented piece is easy to find and relatively hard to forget about (e.g. by attaching some greppable tag to this piece of code and writing with caps at the top that user may want to perform some edits prior to using this config).

Reply:
Done.

Reply:
Cool :+1: